runtime/calamari/src/currency.rs

```diff
@@ -25,3 +25,39 @@ pub const uKMA: Balance = KMA / 1_000_000; // 6 decimal, micro-MA
 pub const fn deposit(items: u32, bytes: u32) -> Balance {
     items as Balance * 15 * mKMA + (bytes as Balance) * 6 * mKMA // TODO: revisit the storage cost here
 }
+
+use pallet_parachain_staking::{BalanceOf, InflationInfo};
+pub fn inflation_config<T: frame_system::Config + pallet_parachain_staking::Config>(
+) -> InflationInfo<BalanceOf<T>> {
+    use pallet_parachain_staking::inflation::Range;
+    use sp_arithmetic::Perbill;
+    use sp_runtime::{traits::UniqueSaturatedInto, PerThing};
+
+    fn to_round_inflation(annual: Range<Perbill>) -> Range<Perbill> {
+        use pallet_parachain_staking::inflation::{
+            perbill_annual_to_perbill_round, BLOCKS_PER_YEAR,
+        };
+        perbill_annual_to_perbill_round(
+            annual,
+            // rounds per year
+            BLOCKS_PER_YEAR / crate::get!(pallet_parachain_staking, DefaultBlocksPerRound, u32),
```

Comment:
Why don't use `Get` directly?
```rust
BLOCKS_PER_YEAR / crate::DefaultBlocksPerRound::get()
```
Then that macro `get!` is unnecessary.

Reply:
good point, moonbeam use it i a lot of tests so it could be useful if we wanted to use it that way too, but i got rid of it